cypress/plugins/sliding-sync/index.ts

```diff
@@ -22,7 +22,8 @@ import { dockerExec, dockerIp, dockerRun, dockerStop } from "../docker";
 import { getFreePort } from "../utils/port";
 import { SynapseInstance } from "../synapsedocker";
 
-// A cypress plugins to add command to start & stop https://github.com/matrix-org/sliding-sync
+// A cypress plugin to add command to start & stop https://github.com/matrix-org/sliding-sync
+// SLIDING_SYNC_PROXY_TAG env used as the docker tag to use for `ghcr.io/matrix-org/sliding-sync-proxy` image.
```

Comment:
shouldn't this be at the top of the file? It's a bit lost here.

Reply:
If you put it above the copyright then it'll fail lint

Reply:
I humbly submit that both the linter and your IDE are braindead.